wagtail_bucketav/testapp/migrations/0001_initial.py

```diff
@@ -0,0 +1,265 @@
+# Generated by Django 4.0.7 on 2022-08-24 13:32
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import taggit.managers
+from wagtail import VERSION as WAGTAIL_VERSION
+import wagtail.images.models
+
+try:
```

Comment:
I've been corrected a few times when I used this try accept set up, better to use wagtail.version check.

It seems that it's better to see the error rather than mask it.